tests/psimulate/redis_dbs/test_registry.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from vivarium_cluster_tools.psimulate.redis_dbs.registry import RegistryManager
+
+
+@pytest.mark.parametrize("num_queues", [1, 5, 7, 10, 25, 50])
+@pytest.mark.parametrize("num_jobs", [1, 10, 1000, 2005, 10000, 20000])
```

Comment:
How long does this test take to run? Do we really need such large numbers? If this is because of having 1000 jobs per queue, why not mock out that number and replace it with something smaller? That actually seems safer anyway, because the numbers you've chosen for test cases are based on that and would potentially not be as interesting if the DEFAULT_JOBS_PER_QUEUE were to change.

Reply:
It takes about 2s when I run it. I didn't specifically choose the numbers based on the 1000 jobs per queue thing--#185 removed that and replaced it with a number of workers per queue, so queues now could have an arbitrary number of jobs. I just chose numbers that seemed like plausible real-world values. It probably doesn't need to include the 20000, i just chose the largest number I could that still took a few seconds to run.

Reply:
removed some large values and added some smaller "pathological" ones